components/mdx/Li.js

```diff
@@ -0,0 +1,5 @@
+export default function Li(props){
+    return (
+        <li className="w-full break-all" {...props}></li>
+    )
+}
```

Comment:
I don't think this is following our coding style

```suggestion
export default function Li(props) {
  return <li className="w-full break-all" {...props}></li>;
}
```

Reply:
Thank you for your suggestion. I'll try to do better next time.